### ISSUE TITLE
Series of exceptions when last opened file is from Dropbox

### ISSUE THREAD

**Issue by hgohel** [authpass/authpass#126]
AuthPass v1.7.3, Windows 10

The last file opened in AuthPass was opened from Dropbox
When I launch AuthPass I see it as the only file in the list and when I click on it to open it, I get a series of exceptions (any typos are mind):

`Unexpected error: MissingPluginException(No implementation found for method init on channel biometric_storage)`

```
Error when Loading file...
MissingPluginException(No implementation found for method init on channel biometric_storage)
```

```
Error while opening file...
Unable to open Instance of 'DiagnosticableNode<Diagnosticable>'.
MissingPluginException(No implementation found for method init on channel biometric_storage)
```

Needless to say, the database isn't opened.  I expected it would ask me for my Dropbox credentials and open the file.

**Comment by hgohel** [comment]
I also get the first error when I click on "Load from Dropbox".  clicking Ok takes me to a screen with "Login to Dropbox" .

Perhaps the problem here is that I had v1.7.2 when I opened this particular kdbx file and prior to installing 1.7.3, I uninstalled 1.7.2.  Did that remove any connection to Dropbox that may have been required to open the file properly?

**Comment by hpoul** [comment]
> I expected it would ask me for my Dropbox credentials and open the file.

Nope, the dropbox auth token should be stored in the windows credential store.. (the biometric storage plugin should do that. On other platforms it uses fingerprint and face id also for optionally remembering master passwords (ie quick unlock). Which is not supported in windows). I assume there is a problem with initialisation of that plugin on windows, I'll have to take a closer look.

I guess you get a similar error when trying to open it again manually through dropbox?

**Comment by hpoul** [comment]
If you have some time maybe you can give this one a try: https://data.authpass.app/data/artifacts/AuthPass-setup-1.7.3_1368.exe

**Comment by hgohel** [comment]
> I guess you get a similar error when trying to open it again manually through dropbox?
Yes, similar error when attempting to load again via Dropbox, but I can click Ok which takes me to a screen with "Login to Dropbox" and then I can login and open the file.  And yes, I did see an entry for AuthPass in Windows Credential Manager.

The version I have is 1.7.3_1368, so no change in behavior after I downloaded the one from the link you provided.

**Comment by hpoul** [comment]
sorry, I somehow didn't start the right build: https://data.authpass.app/data/artifacts/AuthPass-setup-1.7.3_1375.exe
